apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/completionWorks.js

```diff
@@ -79,6 +80,8 @@ const generateTicketDocumentOfCompletionWorks = async ({ task, baseAttrs, contex
         return acc
     }, [])
 
+    const unitType = get(ticket, 'unitType') ?? ''
```

Comment:
Use `flat` for default value
```suggestion
    const unitType = get(ticket, 'unitType') || 'flat'
```